README.md

```diff
@@ -57,178 +27,62 @@ Up-to-date documentation for the MongooseIM master branch can be found on ReadTh
-
-    or
-
-        $ ./rebar get-deps
-        $ ./rebar compile
-
-    To generate full MongooseIM release (with mysql, pgsql or other deps):
-
-        $ make rel
-
-    If more advanced release is required (with some specific db support only,
-    f.e. mysql or pgsql) or you want to set `prefix` or `user` for the
-    installation script please refer to the
-    [release configuration](doc/user-guide/release_config.md)
-    page in our documentation
-
-    `make rel` commands will generate a self-contained OTP system image in the
-    project's `rel/mongooseim` subdirectory. The contents of that directory are as
-    follows:
-    *   `rel/mongooseim/bin` - startup/administration scripts,
-    *   `rel/mongooseim/etc` - configuration files,
-    *   `rel/mongooseim/lib` - MongooseIM binary, header and runtime files,
-    *   `rel/mongooseim/var` - spool directory,
-    *   `rel/mongooseim/log` - log file directory,
-    *   `rel/mongooseim/releases` - release files directory.
-
-3.  Running MongooseIM.
-
-    To run MongooseIM from the project tree after compiling it, change
-    to `$REPO/rel/mongooseim`.
+Download MongooseIM server packages
+-----------------------------------
 
-    There you can use the `mongooseim` command line administration script to
-    start and stop MongooseIM. For example:
-
-        $ bin/mongooseim start
-
-    will start the server.
-
-    You can also run the server in interactive mode:
-
-        $ bin/mongooseim live
-
-    There's also a tool called `mongooseimctl` allowing you to perform some
-    operations on a running instance, e.g.:
-
-        $ bin/mongooseimctl status
-        MongooseIM node mongooseim@localhost:
-          operating system pid: 86026
-          Erlang VM status: started (of: starting | started | stopping)
-          boot script status: started
-          version: 1.6.2-61-g48b8332
-          uptime: 1:12:46
-          logs:
-            log/ejabberd.log
-
-4.  Building the testing target and running tests.
-
-    For testing purposes there's a different make target available:
-
-        $ make devrel
-
-    which will generate releases in `$REPO/dev/` and prepare
-    them for testing and generating coverage reports.
-
-    To run the tests (from project's root directory, i.e. `$REPO`):
-
-        $ dev/mongooseim_node1/bin/mongooseim start
-        $ dev/mongooseim_node2/bin/mongooseim start
-        $ make quicktest
-
-    The test results will show up in the console`.
-
-
-Test suite
-----------
+For a quick start just
+[download the pre-built package](https://www.erlang-solutions.com/resources/download.html)
+that suits your platform: Ubuntu, Debian, CentOS, and macOS.
 
-In order to test and validate your XMPP servers, here are useful tools:
-* [escalus](https://github.com/esl/escalus): Erlang XMPP client
-* [amoc](https://github.com/esl/amoc): a load testing tools
+An _experimental_ Docker image exists on: https://registry.hub.docker.com/u/mongooseim/mongooseim-docker/
```

Comment:
Let's use the new docker image: https://hub.docker.com/r/mongooseim/mongooseim/ and its repo here: https://github.com/esl/mongooseim-docker